ml_examples/llama31/README.md

```diff
@@ -0,0 +1,20 @@
+
+# Run inference on Verily Workbench using the Llama 3.1 8B model
+
+The [vwb_8b_v100_llama31_hf.ipynb](./vwb_8b_v100_llama31_hf.ipynb) notebook example shows how to
+load and run inference with a [Llama 3.1
+model](https://llama.meta.com/docs/model-cards-and-prompt-formats/llama3_1/) on Verily Workbench,
+using the [Huggingface](https://huggingface.co/) libraries and model access.
+
+You'll need a Huggingface [account](https://huggingface.co/join) and [access
+token](https://huggingface.co/settings/tokens). You'll also need to apply for *approval to access
```

Comment:
Do you know what permissions are required for the token? For example, if a read-only token is sufficient, that would be good to mention.

Reply:
Yes, read-only is sufficient, I will add that.